pom.xml

```diff
@@ -12,6 +12,8 @@
   <url>http://openrefine.org/</url>
 
   <modules>
+        <module>or-model</module>
+        <module>or-testing</module>
```

Comment:
You could merge it with `or-model`, but I think it would be a bit sad to do that, since `or-model` would then have to depend on TestNG / Mockito and the like. The goal of this artifact is to gather all testing utilities that modules in OpenRefine or in extensions can rely on, without having to copy and paste those into their own module.